common_rest/src/main/java/uk/gov/dhsc/htbhf/requestcontext/RequestContextConfiguration.java

```diff
@@ -77,7 +80,9 @@ public ObjectMapper objectMapper() {
         return new ObjectMapper()
                 .findAndRegisterModules()
                 .registerModule(javaTimeModule)
-                .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+                .configure(MapperFeature.DEFAULT_VIEW_INCLUSION, false)
+                .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+                .configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
```

Comment:
According to the Spring documentation, these values are the defaults, so any reason why you needed to add them?

Reply:
This isn't spring managed. I've updated the comment to make that clearer.